### DISCUSSION TITLE
JUNO第二阶段的大作业为什么大部分type=1和-1的波形集中在920左右

### DISCUSSION THREAD

**Discussion by Lodfy-Feng** [physics-data/faq#170] [Q&A] [answered]
不太清楚PMT怎么判定接受到一个光子的，看起来信号近似一条直线。如果是考虑底噪有900多，感觉也不太合理。

**Comment by heroxbd** [comment, marked as answer]
请 @Berrysoft 来确认

约 920 是所谓的“基线”，即整个电路默认不是零电位的。信号是偏离这个基线的电压脉冲。

**Reply by Berrysoft** [reply]
是的，这是基线。是约为920的值，但是其准确值并不是920。
